code/_helpers/mobs.dm

```diff
@@ -275,8 +279,24 @@ Proc for attack log creation, because really why not
 
 	return FALSE
 
+/proc/is_excelsior(var/mob/M)
+	var/obj/item/weapon/implant/excelsior/E = locate(/obj/item/weapon/implant/excelsior) in M
+	if (E && E.wearer == M)
+		return TRUE
+
+	return FALSE
+
 /proc/mob_hearers(var/atom/movable/heard_atom, var/range = world.view)
 	. = list()
 
 	for(var/mob/hmob in hearers(range, heard_atom))
 		. |= hmob
+
+
+// Returns a bitfield representing the mob's type as relevant to the devour system.
+/mob/proc/find_type()
```

Comment:
May be `get_types()` would be better name?

Reply:
I changed it to classification, since type is an overused term